docs/Documentation/Configuration/Version-Changes/Migration.md

```diff
@@ -321,3 +322,67 @@ castMMOItemSkill: "mmoskill DEEP_WOUND trigger:RIGHT_CLICK"
 ```
 
 </div>
+
+### 2.0.0-DEV-XXX - Static Event Rework
+
+Until now if you used non-static events in a [schedule](../../Scripting/Schedules.md), they were executed for every player that was online at the time.  
+If you run such a schedule now you will get a warning message in the console similar to this:
```

Comment:
I am not sure if this could be improved so the people may understand this better, but for me its ok, and i would merge this then.